core/domain/dogodek.py

```diff
@@ -10,6 +12,14 @@ class TipDogodka(str, Enum):
 @dataclass
 class Dogodek:
 	ime: str
+	cena: float
 	tip: TipDogodka
 	zacetek: datetime
 	konec: datetime
+	uporabnik: Uporabnik
+	placnik: Uporabnik
+	placano: bool
```

Comment:
V dogodku koledarja neveš ali je uporabnik plačal. Pa vse informacije iz koledarja lahko flikneš v uporabnika.

Reply:
Hmm sem pustil kar tko  (odstranil sem samo placano: , ker to imam že v knjižnem vnosu), ker rabim pri ustvarjanju poročila podatek kdaj je bil dogodek ustvarjen, da ga lahko shranim pod itsti datum. 
		```
	for dogodek in google_dogodki:
				if dogodek.zacetek.date() == dd:
```
Če se prav razumeva 🤔

Reply:
@ethernal12 Če pogledaš class Uporabnik se notri nahajajo naslednje stvari...
```
class Uporabnik:
	ime: str
	priimek: str
	email: str
	email_racuna: str
	telefon: str
```
V primeru če boš imel v dogodku 2 uporabnika (uporabnik in plačnik) nevem kje boš dobil vse te informacije iz eventa za plačnika ki se nahaja v koledarju... Kolikor je meni znano če pogledam koledar event je podano...

- Name
- Phone
- Email
- Price
- Email osebe, ki bo plačala inštrukcije

In vse informacije iz eventa lahko postaviš v enega uporabnika in mislim da bo to zadostovalo?

Reply:
Ok, se nisva razumela, sem popravil 👍